src/current/_includes/v24.3/physical-replication/fast-failback-syntax.md

```diff
@@ -1,4 +1,4 @@
-To fail back to a cluster that was previously the primary cluster, use the [`ALTER VIRTUAL CLUSTER`]({% link {{ page.version.version }}/alter-virtual-cluster.md %}) syntax:
+To {% if page.name == "alter-virtual-cluster.md" %} [fail back]({% link {{ page.version.version }}/failover-replication.md %}#fail-back-to-the-primary-cluster) {% else %} fail back {% endif %} to a cluster that was previously the primary cluster, use the {% if page.name == "alter-virtual-cluster.md" %} `ALTER VIRTUAL CLUSTER` {% else %} [`ALTER VIRTUAL CLUSTER`]({% link {{ page.version.version }}/alter-virtual-cluster.md %}) {% endif %} syntax:
```

Comment:
looks like this leaked into this pcr read from standby diff. i don't care, but just calling out if that was a mistake.

Reply:
Oh this was just a small update I included to the logic of the page, so we're linking ti one page if a read is on x and another page if the reader is on y